packages/react-native/src/Authenticator/common/DefaultHeader/DefaultHeader.tsx

```diff
@@ -5,6 +5,12 @@ import { DefaultHeaderProps } from './types';
 
 export default function DefaultHeader({
   children,
-}: DefaultHeaderProps): JSX.Element {
-  return <Heading level={3}>{children}</Heading>;
+  level = 3,
+  ...rest
+}: DefaultHeaderProps): JSX.Element | null {
+  return children ? (
```

Comment:
Only render the `Heading` when a child is provided to prevent whitespace from being rendered